packages/fluentui/react-builder/src/components/ComponentTree.tsx

```diff
@@ -160,6 +198,23 @@ export const ComponentTree: React.FunctionComponent<ComponentTreeProps> = ({
     );
   };
 
+  const handleAddComponent = React.useCallback(
+    (uuid, where) => {
+      onAddComponent?.(uuid, where);
+    },
+    [onAddComponent],
+  );
+
+  const handleDeleteComponent = React.useCallback(
+    uuid => {
+      onSelectComponent(jsonTreeFindElement(tree, uuid));
+      setTimeout(() => {
```

Comment:
I would rather rename `onDeleteComponent` to `onDeleteSelectedComponent` and add `onDeleteComponent(uuid)`.